config/pure-python-with-multiprocess/packages.txt

```diff
@@ -0,0 +1,4 @@
+# Packages configured for the pure python version are listed here.
```

Comment:
"for the pure python version" *with multiprocessing*, perhaps?

Reply:
I renamed it to `buildout recipe`.